v2/policyfile.go

```diff
@@ -116,22 +128,31 @@ func (pr *PolicyFileResource) Get(ctx context.Context) (*ACL, error) {
 		return nil, err
 	}
 
-	return body[ACL](pr, req)
+	acl, header, err := bodyWithResponseHeader[ACL](pr, req)
+	if err != nil {
+		return nil, err
+	}
+	acl.ETag = header.Get("ETag")
```

Comment:
If you spell it the Go canonical way (Etag) here you avoid an alloc.

Reply:
TIL, thanks!